packages/frontend-v2/components/Error/ConfirmEmailWarningModal.tsx

```diff
@@ -0,0 +1,74 @@
+import {
```

Comment:
The current implementation of this modal uses localStorage to find out if it is the user's first time visiting this page with the `alreadyVisited` key. However, because next loads this page before localStorage can be accessed, the default value is null, and so the component thinks that the user HAS visited the page before, and thus the warning modal pops up.